src/bidiMapper/domains/script/scriptEvaluator.ts

```diff
@@ -319,6 +65,205 @@ export async function stringifyObject(
 }
 
 export class ScriptEvaluator {
+  async deserializeToCdpArg(
```

Comment:
1. `#deserializeToCdpArg`, as it is used only in the class.
2. Why not to move the other block as well?
```
const CALL_FUNCTION_STACKTRACE_LINE_OFFSET = 1;
const EVALUATE_STACKTRACE_LINE_OFFSET = 0;
export const SHARED_ID_DIVIDER = '_element_';
function cdpRemoteObjectToCallArgument(
  cdpRemoteObject: Protocol.Runtime.RemoteObject
): Protocol.Runtime.CallArgument {
  if (cdpRemoteObject.objectId !== undefined) {
    return {objectId: cdpRemoteObject.objectId};
  }
  if (cdpRemoteObject.unserializableValue !== undefined) {
    return {unserializableValue: cdpRemoteObject.unserializableValue};
  }
  return {value: cdpRemoteObject.value};
}
```

Reply:
Staged in https://github.com/GoogleChromeLabs/chromium-bidi/pull/319

Reply:
2) Likewise, staged in https://github.com/GoogleChromeLabs/chromium-bidi/pull/319

Reply:
Consider making it in a dedicated PR